src/sentry/static/sentry/app/views/alerts/rules/details/timeline.tsx

```diff
@@ -248,8 +216,8 @@ type Props = {
 class Timeline extends React.Component<Props> {
   renderEmptyMessage = () => {
     return (
-      <EmptyStateWarning small withIcon={false}>
-        {t('No alerts have been triggered yet')}
+      <EmptyStateWarning small withIcon={false} noPadding>
```

Comment:
is it possible to have zero padding using a styled component instead of having to add a new prop to `EmptyStateWarning` or is the prop preferred because we will likely have this in other places?

Reply:
I think using a styled component would be better, just hadn't thought of it